api/foxpass_create_users.py

```diff
@@ -30,12 +38,18 @@ def main():
             try:
                 email = row[0]
                 username = row[1]
+                password = row[2]
```

Comment:
what happens on line 46 if password isn't included

Reply:
if they don't have "" there the same thing that happens if email or username isn't included.  IndexError and it continues.  Line 46 won't be run.

Reply:
is that expected behavior? i.e. if I have a file with just email/username will this script work?

Reply:
I should say, either a trailing `,` or `,""` it will resolve False and not update the password in our system.  If it's omitted, it's a bad line and we skip it.